src/server/api/session.post.ts

```diff
@@ -14,7 +14,17 @@ export default defineEventHandler(async (event) => {
       statusMessage: 'Invalid state',
     });
   }
-  if (!isPasswordValid(password, PASSWORD_HASH)) {
+
+  const users = await Database.getUsers();
+  const user = users.find((user) => user.username == username);
+  if (!user)
+    throw createError({
+      statusCode: 400,
+      statusMessage: 'User with username does not exist',
+    });
```

Comment:
this should be made more secure. this would allow bad actors to probe correct usernames.
also timing should be considered